apps/wallet/src/ui/app/shared/forms/FormLabel.tsx

```diff
@@ -6,18 +6,20 @@ import { Text } from '_app/shared/text';
 import type { ReactNode } from 'react';
 
 export type FormLabelProps = {
-	label: ReactNode;
+	label?: ReactNode;
 	children: ReactNode;
 };
 
 export function FormLabel({ label, children }: FormLabelProps) {
 	return (
 		<label className="flex flex-col flex-nowrap gap-2.5">
-			<div className="pl-2.5">
-				<Text variant="body" color="steel-darker" weight="semibold">
-					{label}
-				</Text>
-			</div>
+			{label && (
```

Comment:
This is fine for now, but we want to make sure all of our form inputs are accessible which this doesn't encourage. To be honest, I think the form primitives I built need a little more thought or really we should just copy + pasta from https://ui.shadcn.com/examples/forms (it turns out creating proper form abstractions is a hard thing to do 😛)